frontend/src/scenes/events/PropertiesVolumeTable.tsx

```diff
@@ -69,17 +71,19 @@ export function PropertiesVolumeTable(): JSX.Element {
             </Button>
             <br />
             <br />
-            {user?.team.event_properties_with_usage[0]?.volume === null && (
+            {user?.team?.event_properties_with_usage[0]?.volume === null && (
                 <>
                     <Alert
                         type="warning"
                         description="We haven't been able to get usage and volume data yet. Please check back later"
+                        message={<></>}
                     />
                     <br />
                 </>
             )}
             <Table
-                dataSource={user?.team.event_properties_with_usage
+                className="ph-no-capture"
```

Comment:
Added a comment in the main issue about whether to capture property names. Regardless, I think it's worth sanitizing each column instead of the whole table, for recordings to be more useful.